src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -116,7 +116,7 @@
       }
 
       .institution-information {
-        background-color: #f1f1f1;
+        background-color: $color-gray-lightest;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L71